src/GitVersionCore/IncrementStrategyFinder.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace GitVersion
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using VersionCalculation.BaseVersionCalculators;
+    using LibGit2Sharp;
+
+    public enum CommitMessageVersionMode
```

Comment:
It allows the user to either disable the commit message parsing, or to limit it to only look at merge messages. Limiting to merge messages may be useful for repo maintainers who want to specify the severity themselves when merging a pull request.